draft-ietf-capport-architecture.xml

```diff
@@ -695,7 +695,29 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 link. In such cases, identification could be performed by subnet, such as the /64
                 to which the IP belongs.
             </t>
-           </section>
+         </section>
+         <section anchor="context_free_url" title="Context-free URL">
+           <t>
+             The URLs provided in the API SHOULD contain all the information necessary to render
+             the resources requested: they should be context free so it does not depend on ambient
```

Comment:
As phrased, this is a bit ambiguous. The resource should be context free, not the URL -- i.e. all context should be in the URL, and the method for generating the resource should not depend on anything but the context within that URL. In my first read of this section, I took this to mean that the UR: should be context free.

Perhaps rephrase to:

"...the resources requested: the resources should be context free so they do not ..."

Reply:
how about:
...the resources requested: the resources should not depend on ambient...

Reply:
Thanks. That sounds good.